app/src/main/res/layout/fragment_amortisationsrechnerpv.xml

```diff
@@ -293,6 +286,66 @@
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/amortPV_edit_measure_margin_top"
+                    android:text="@string/text_euro"
+                    android:textSize="@dimen/content_text_normal"
+                    app:layout_constraintStart_toEndOf="@+id/amortPV_edit_anschaffung"
+                    app:layout_constraintTop_toTopOf="@+id/amortPV_edit_anschaffung" />
+
+                <TextView
+                    android:id="@+id/amortPV_text_ertrag_kwh_pr_kwp"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/amortPV_edit_measure_margin_top"
+                    android:text="@string/text_kWh_pro_kWp"
+                    android:textSize="@dimen/content_text_normal"
+                    app:layout_constraintStart_toEndOf="@+id/amortPV_edit_ertrag"
+                    app:layout_constraintTop_toTopOf="@+id/amortPV_edit_ertrag" />
+
+                <TextView
+                    android:id="@+id/amortPV_text_vergütung_cent"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/amortPV_edit_measure_margin_top"
+                    android:text="@string/text_cent"
+                    android:textSize="@dimen/content_text_normal"
+                    app:layout_constraintStart_toEndOf="@+id/amortPV_edit_vergütung"
+                    app:layout_constraintTop_toTopOf="@+id/amortPV_edit_vergütung" />
+
+                <TextView
+                    android:id="@+id/amortPV_text_eigenverbrauch_prozent"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/amortPV_edit_measure_margin_top"
+                    android:text="@string/text_prozent"
+                    android:textSize="@dimen/content_text_normal"
+                    app:layout_constraintStart_toEndOf="@+id/amortPV_edit_eigenverbrauch"
+                    app:layout_constraintTop_toTopOf="@+id/amortPV_edit_eigenverbrauch" />
+
+                <TextView
+                    android:id="@+id/amortPV_text_preis_kwh_cent"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textSize="@dimen/content_text_normal"
+                    android:layout_marginTop="@dimen/amortPV_edit_measure_margin_top"
+                    android:text="@string/text_cent"
+                    app:layout_constraintStart_toEndOf="@+id/amortPV_edit_preis_kwh"
+                    app:layout_constraintTop_toTopOf="@+id/amortPV_edit_preis_kwh" />
+
             </androidx.constraintlayout.widget.ConstraintLayout>
 
```

Comment:
A lot of XML naming and ids should be in snake case only.

Reply:
This will be fixed in the Pull Request from AmortisationsrechnerPV.